classes/core/AltapayOrderStatus.php

```diff
@@ -52,80 +45,77 @@ public function orderStatusChanged( $orderID, $currentStatus, $nextStatus, $orde
 		$refunded = 0;
 		$status   = '';
 
-		$api = new AltapayMerchantAPI(
-			esc_attr( get_option( 'altapay_gateway_url' ) ),
-			esc_attr( get_option( 'altapay_username' ) ),
-			esc_attr( get_option( 'altapay_password' ) ),
-			null
-		);
-		try {
-			$api->login();
-		} catch ( Exception $e ) {
-			$_SESSION['altapay_login_error'] = $e->getMessage();
+		$settings = new Core\AltapaySettings();
+		$api      = $settings->altapayApiLogin();
```

Comment:
Probably rename `$api` here to `$login` so that the same variable isn't being reused for a different type on line 57

Reply:
I still see a lot of this all involving `altapayApiLogin()`, probably search for each all usages and change the variable to $login